src/common/API.elm

```diff
@@ -296,6 +301,43 @@ lookupTrustee onError onSuccess email =
     |> Task.mapError httpErrorToString
     |> Task.perform onError onSuccess
 
+------------
+-- PLACES --
+------------
+
+addPlace : (String -> a) -> (List Place -> a) -> Cmd a
+addPlace onError onSuccess =
+  (Place.encoder Place.createEmpty)
+    |> Encode.encode 0
+    |> Http.string
+    |> post'
+      (Decode.list Place.decoder) ("/api/secure/postLocation" )
+    |> Task.mapError httpErrorToString
+    |> Task.perform onError onSuccess
+
+fetchPlaces : (String -> a) -> (List Place -> a) -> Cmd a
+fetchPlaces onError onSuccess =
+  Http.get (Decode.list Place.decoder) ("/api/secure/getLocation")
+  |> Task.mapError httpErrorToString
+  |> Task.perform onError onSuccess
+
+updatePlace : (String -> a) -> (Place -> a) -> Place -> Cmd a
+updatePlace onError onSuccess place =
+  (Place.encoder place)
+    |> Encode.encode 0
+    |> Http.string
+    |> post'
+      Place.decoder ("/api/" ++ toString place.id ++ "/updateLocation" )
```

Comment:
if you take a look at some of the other api calls, you'll see that we're either using openEndpoint or secureEndpoint to generate the urls; can we switch these calls to do the same?